src/main/resources/interface-specification.yml

```diff
@@ -460,6 +460,57 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
+  /eservices/{eServiceId}/descriptors/{descriptorId}/update:
+    parameters:
+      - $ref: '#/components/parameters/CorrelationIdHeader'
+    post:
+      security:
+        - bearerAuth: []
+      tags:
+        - process
+      summary: Publish the selected descriptor.
+      operationId: updatePublishedDescriptor
+      parameters:
+        - name: eServiceId
+          in: path
+          description: the eservice id
+          required: true
+          schema:
+            type: string
+            format: uuid
+        - name: descriptorId
+          in: path
+          description: the descriptor Id
+          required: true
+          schema:
+            type: string
+            format: uuid
+      requestBody:
+        description: Payload
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/UpdateEServicePublishedDescriptorSeed'
```

Comment:
```suggestion
              $ref: '#/components/schemas/UpdateEServiceDescriptorSeed'
```

Reply:
Already existsm are you sure?

Reply:
you're right, let's find another name

Reply:
`UpdateEServiceDescriptorQuotas`?
`UpdateEServiceDescriptorDetails`?